index.js

```diff
@@ -66,6 +69,14 @@ module.exports = class Autobase {
 
   // Private Methods
 
+  _inputsValidation () {
```

Comment:
Can you rename this method to `_validateInputs`?

Reply:
🙈 sorry for this bad name ahah! yeah with pleasure ^^